ldap/install.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+set -e
+
+trap 'ret=$?; test $ret -ne 0 && printf "failed\n\n" >&2; exit $ret' EXIT
+
+log_info() {
+  printf "\n\e[0;35m $1\e[0m\n\n"
+}
+
+source /build/base.config
+
+#------------------------
+# Bootstrap LDAP OUs
+#------------------------
+mkdir -p /container/service/slapd/assets/config/bootstrap/ldif/custom
+cat > /container/service/slapd/assets/config/bootstrap/ldif/custom/0-ous.ldif <<EOF
+dn: ou=People,dc=example,dc=org
+objectClass: organizationalUnit
+ou: People
+
+dn: ou=Groups,dc=example,dc=org
+objectClass: organizationalUnit
+ou: Groups
+EOF
+
+#------------------------
+# Setup user accounts
+#------------------------
+
+idnumber=1001
+for uid in hpcadmin $USERS
+do
+    log_info "Adding LDIF for $uid user account with uidnumber $idnumber.."
+    passvar="PASSWD_$uid"
+    passwd=${!passvar:-ilovelinux}
+    cat > /container/service/slapd/assets/config/bootstrap/ldif/custom/1-$uid.ldif <<EOF
+dn: cn=${uid},ou=People,dc=example,dc=org
+objectClass: person
+objectClass: posixAccount
+objectClass: inetOrgPerson
+gecos: ${uid}
+cn: ${uid}
+sn: ${uid}
+uid: ${uid}
+homeDirectory: /home/${uid}
+uidNumber: ${idnumber}
+gidNumber: ${idnumber}
+mail: ${uid}@example.com
```

Comment:
Can we add loginShell so we get bash as the default shell:
```
loginShell: /bin/bash
```

Reply:
Yep, I made that change after Ben noticed it defaulted to /bin/sh